src/clients/ntp-client_auto.rb

```diff
@@ -63,32 +58,17 @@ def main
         @ret = NtpClient.AutoPackages
       # Change configuration (run AutoSequence)
       elsif @func == "Change"
-        @ret = NtpClientAutoSequence()
+        # TODO: implement
+        Yast::Report.Error("Not supported yet for chrony")
+        @ret = :next
       # Import configuration
       elsif @func == "Import"
-        @ret = NtpClient.Import(@param)
+        Yast::Report.Error("Not supported yet for chrony")
+        @ret = true
```

Comment:
hm, shouldn't it return false (popups can be timed out in some scenarios)

Reply:
no, requirement was to not crash, so it should not complain now ( beside that error message ).